featurebyte/config.py

```diff
@@ -170,9 +170,9 @@ def request(
         """
         try:
             headers = kwargs.get("headers", {})
-            headers["active-catalog-id"] = headers.get(
-                "active-catalog-id", str(get_active_catalog_id())
-            )
+            active_catalog_id = headers.get("active-catalog-id", get_active_catalog_id())
+            if active_catalog_id:
+                headers["active-catalog-id"] = str(active_catalog_id)
```

Comment:
Only include catalog id when a catalog is active, allow requests without active catalog